modules/core/src/main/java/org/apache/ignite/internal/processors/security/GridSecurityProcessor.java

```diff
@@ -133,4 +134,39 @@ public void authorize(String name, SecurityPermission perm, SecurityContext secu
     public default boolean sandboxEnabled() {
         return false;
     }
+
+    /**
+     * Creates user with the specified login and password.
+     *
+     * @param login Login of the user to be created.
+     * @param pwd User password.
+     * @throws IgniteCheckedException If error occurred.
+     */
+    @IgniteExperimental
```

Comment:
We don't need IgniteExperimental for internal classes.

Reply:
Reverted.